intermediate-javascript-calorie-counter/index160.js

```diff
@@ -0,0 +1,26 @@
+document.getElementById('calorie-form').onsubmit = calculate;
+
+function calculate(e) {
+  e.preventDefault();
+
+  const total = Array.from(document.getElementsByClassName('cal-control'))
+    .map(meal => Number(meal.value))
+    .reduce();
+}
+/*
+The reduce method takes a callback function as well as an optional initial value, its function signature is as follows:
```

Comment:
The `reduce()` method takes a callback function with at least two arguments, an accumulator and a current value:
```
function(accumulator, currentValue) { 
  /*code to run*/
}
```
Or using arrow functions:
`(accumulator, currentValue) => { /*code to run*\/}`
Insert the callback function above as an argument in the `.reduce()` function.